pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/region_base.go

```diff
@@ -90,6 +91,7 @@ type QoSRegionBase struct {
 	// the former has higher priority; provisionPolicyNameInUse indicates the provision
 	// policy in-use currently
 	provisionPolicies        []*internalProvisionPolicy
+	provisionUpdatePolicies  []*internalProvisionPolicy
```

Comment:
The naming is confusing. provisionPoliciesInUpdatePriority might be better. Add comment for it.